torch/fft/__init__.py

```diff
@@ -767,10 +809,10 @@
     >>> torch.fft.ifft(t)
     tensor([ 2.0000+-0.0000j, -0.5000-0.6882j, -0.5000-0.1625j, -0.5000+0.1625j,
         -0.5000+0.6882j])
-""")
+""".format(**common_args))
 
 fftfreq = _add_docstr(_fft.fft_fftfreq, r"""
-fftfreq(n, d=1.0, *, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
+fftfreq(n, d=1.0, *, out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
```

Comment:
nit: out= should appear last in the list of kwargs

Reply:
I checked some other factory functions and all of them list `out` before the `TensorOptions` arguements:
- [`arange`](https://pytorch.org/docs/master/generated/torch.arange.html?highlight=arange#torch.arange)
- [`ones`](https://pytorch.org/docs/master/generated/torch.ones.html?highlight=ones#torch.ones)
- [`linspace`](https://pytorch.org/docs/master/generated/torch.ones.html?highlight=ones#torch.ones)

Reply:
oh snap! Cool, thanks for checking